Automattic/jetpack
# Extension: fix typo error country code

Fixes #22170

#### Changes proposed in this Pull Request:

* There was a typo error cyprus country code was written as +537. Correct value is +357

#### Jetpack product discussion NO



#### Does this pull request change what data or activity we track or use? NO



#### Testing instructions:





* After inserting the WhatsApp button block into a page or a post, open up the settings in the right panel. There you will see fields for Phone Number. The first field helps you select a country code - if you search through this list you will see that the country code +357 for Cyprus is present.

#### Screenshots:
| Before changes                                                                                                                                                           | After Changes                                                                                                                                                            |
|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| <img width="704" alt="Screenshot 2021-12-31 at 12 52 36 AM" src="https://user-images.githubusercontent.com/21127788/147782983-4297d86f-3b24-459a-964d-d19b1069ef5b.png"> | <img width="875" alt="Screenshot 2021-12-31 at 12 56 01 AM" src="https://user-images.githubusercontent.com/21127788/147783023-c8c87dde-ccc2-468f-9786-d2be3c27bf70.png"> |
